tests/deploy/cloudbeat-eks-pytest.yaml

```diff
@@ -331,7 +331,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        image: public.ecr.aws/z7e1r9l0/cloudbeat:latest
+        image: 704479110758.dkr.ecr.eu-west-2.amazonaws.com/cloudbeat:latest
```

Comment:
it is better that the container repository address would be handled with gh environments

Reply:
This file is an internal yaml used by pytest infra to execute cloudbeat. Since we decided to move cloudbeat / agent execution from pytest infra to helm deployment, image management will be managed by helm env variables.